### ISSUE TITLE
Implement Neural Network blocks

### ISSUE THREAD

**Issue by arjun-rao** [miloide/MiloServer#111]
## Expected Behaviour
* Allow creation of neural networks ( Support up to MNIST requirements) 
* Block design similar to plots - with statement inputs that allow chaining of layers
* Visualize Dense layers 
* Visualize training loss and accuracy
